packages/forms/docs/01-installation.md

```diff
@@ -39,14 +39,15 @@ You're now ready to start [building forms](getting-started)!
 The package uses the following dependencies:
 
 - [Alpine.js](https://alpinejs.dev)
+- [Alpine Floating UI](https://github.com/awcodes/alpine-floating-ui)
 - [Tailwind CSS](https://tailwindcss.com)
 - [Tailwind CSS Forms plugin](https://github.com/tailwindlabs/tailwindcss-forms)
 - [Tailwind CSS Typography plugin](https://tailwindcss.com/docs/typography-plugin)
 
 You may install these through NPM:
 
 ```bash
-npm install alpinejs tailwindcss @tailwindcss/forms @tailwindcss/typography --save-dev
+npm install alpinejs @awcodes/alpine-floating-ui tailwindcss @tailwindcss/forms @tailwindcss/typography --save-dev
```

Comment:
Breaking change?

Reply:
Yeah, just changed it to how we'd ideally have it for now.
We'll have to discuss how we'd like to actually handle this before merging the PR...

Reply:
So what would you suggest, @danharrin? Importing Alpine Floating UI inside the forms package results in duplicate bundled code if you also manually import it elsewhere.

Reply:
It needs to stay in the forms package until v3